src/XMLHttpRequest/XMLHttpRequest/createXMLHttpRequestOverride.ts

```diff
@@ -105,10 +105,10 @@ export const createXMLHttpRequestOverride = (
       this.withCredentials = false
       this.status = 200
       this.statusText = 'OK'
-      this.user = ''
-      this.password = ''
+      this.user = undefined
+      this.password = undefined
       this.data = ''
-      this.async = true
+      this.async = undefined
```

Comment:
Could you please leave an explanatory comment why the default value of `this.async` is `undefined`, while when it's used it's fallbacked to `true`? I remember you've found a reason behind this, but it needs to be noted in the code to prevent any confusion. Thanks.

Reply:
There is no real reason - actually since it's type as optional, I suggest to remove the initialization all together. It only makes sense to have a value after open() I would say.